framework/src/modules/chain/submodules/peers.js

```diff
@@ -55,7 +55,7 @@ class Peers {
 			channel: scope.channel,
 			blocks: scope.modules.blocks,
 			constants: {
-				minBroadhashConsensus: scope.config.constants.MIN_BROADHASH_CONSENSUS,
+				minBroadhashConsensus: scope.config.constants.minBroadhashConsensus,
```

Comment:
Why do we keep `scope` in this file?

Reply:
I didn't want to touch so much because #3806

Reply:
Ok can we create an issue to remove it?

Reply:
I think #3808 should solve this issue 😉